report-generator/src/main/kotlin/com/teamscale/report/util/FileSystemUtils.kt

```diff
@@ -0,0 +1,32 @@
+package com.teamscale.report.util
+
+import java.io.File
+import java.io.IOException
+
+object FileSystemUtils {
+
+    /** Unix file path separator  */
+    val UNIX_SEPARATOR = '/'
```

Comment:
i've seen at least one other place where this char literal is cloned. plesae find and reuse

Reply:
It's used in a few places, but with different semantics (except the javaws occurrence).
It's used as test path separator and as separator for package name elements in other places.